### PR TITLE
fix USRBG & ViewIcons in new profiles

### PR DESCRIPTION
**Pull request by AutumnVN**

### CODE REVIEW

File: src/plugins/usrbg/index.tsx [hunk 1]
```diff
@@ -75,16 +71,18 @@ export default definePlugin({
         },
         {
             find: /overrideBannerSrc:\i,overrideBannerWidth:/,
```
Comment (Vendicated) [inline src/plugins/usrbg/index.tsx:73]: is this patch even still needed? according to report this find doesnt find any module

Comment (AutumnVN) [reply src/plugins/usrbg/index.tsx]: fixed, i forgot that

File: src/plugins/usrbg/index.tsx [hunk 2]
```diff
@@ -63,28 +63,26 @@ export default definePlugin({
...
+            replacement: {
+                match: /(\i)\.premiumType/,
+                replace: "$self.premiumHook($1)||$&"
+            }
+        },
+        {
+            find: "BannerLoadingStatus:function",
+            replacement: {
+                match: /(?<=void 0:)\i.getPreviewBanner\(\i,\i,\i\)/,
+                replace: "$self.useBannerHook(arguments[0])||$&"
```
Comment (Vendicated) [inline src/plugins/usrbg/index.tsx:83]: this breaks the rules of hooks. hooks must not be called conditionally

image

Comment (Vendicated) [reply src/plugins/usrbg/index.tsx:83]: oh... this is not a react hook

**Comment by Vendicated** [comment]
thanks :3

**Pull request merged**